src/pages/index.tsx

```diff
@@ -243,20 +243,24 @@ const Home: NextPage = () => {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <Layout>
-        <h1 className="mb-16 mt-8 text-8xl font-extrabold tracking-tight">
-          League
-        </h1>
-        <div className="absolute right-10">
-          <Boxer />
-        </div>
-        <div className="absolute left-10 rotate-180">
-          <Boxer />
-        </div>
-        <div className="absolute left-1/2 top-1/3 -translate-x-60 -translate-y-40">
-          <Run />
-        </div>
-        <div className="absolute left-1/2 top-1/2 -translate-y-60 translate-x-40">
-          <Pullup />
+        {/* <h1 className="mb-16 mt-8 text-8xl font-extrabold tracking-tight">
+           League
+         </h1>
+         <div className="absolute right-10">
+           <Boxer />
+         </div>
+         <div className="absolute left-10 rotate-180">
+           <Boxer />
+         </div>
+         <div className="absolute left-1/2 top-1/3 -translate-x-60 -translate-y-40">
+           <Run />
+         </div>
+         <div className="absolute left-1/2 top-1/2 -translate-y-60 translate-x-40">
+           <Pullup /> */}
+        <div className="effect-container">
+          <h1 className="effect dark:text my-12 font-league text-7xl md:text-9xl">
+            Le<span className="pl-1">a</span>gue
```

Comment:
toto je naco?

Reply:
To je můj custom kerning - na úpravu mezery mezi jednotlivými znaky v písmu, aby bylo opticky vyvážené.